recipes/template/README.md

```diff
@@ -0,0 +1,87 @@
+# Earth2Studio Recipe Template
+
+This is a template designed to be boiler plate.
+It is expected all recipes follow the same template with best effort.
```

Comment:
Maybe 'All recipes should follow this template as closely as possible'

Reply:
Fixed, thanks!